packages/front-end/components/Forms/CodeTextArea.tsx

```diff
@@ -42,23 +42,32 @@ export default function CodeTextArea({
   // eslint-disable-next-line
   const fieldProps = otherProps as any;
 
+  const semicolonWarning =
+    "Warning: Please remove any terminating semicolons. GrowthBook uses Common Table Expressions that will break from terminating semicolons.";
+
+  if (language === "sql" && value.includes(";")) {
```

Comment:
This does overwrite any existing error messages. As of right now, none of the `CodeTextArea` components using SQL are using the `error` prop, so it shouldn't be an issue, but I wanted to call it out. 

I'm adding the error here as I can do so in a single spot, as opposed to having to make duplicate changes on each of the forms consuming the `CodeTextArea` component. 

Open to feedback, though.